components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -968,8 +965,8 @@ open class FxaAccountManager(
             type = deviceConfig.type.intoSyncType()
         ))
 
-        // If device supports SEND_TAB...
-        if (deviceConfig.capabilities.contains(DeviceCapability.SEND_TAB)) {
+        // If device supports SEND_TAB, and we're not recovering from an auth problem...
+        if (deviceConfig.capabilities.contains(DeviceCapability.SEND_TAB) && authType != AuthType.Recovered) {
```

Comment:
This seems sensible from reading the ticket. Would this cause the device constellation we have to be invalid on genuine recovered states vs. 401 error states?

Reply:
I don't think so - other devices are unlikely to have changed just because your current one just recovered from a 401. Also - we will refresh the constellation next time user tries to "sync now", and during a startup, so even if it is out of date a bit, that's not for long.